tools/manifest-templator/manifest-templator.go

```diff
@@ -136,6 +136,7 @@ func getCNA(data *templateData) {
 
 	// Get CNA Deployment
 	cnadeployment := components.GetDeployment(
+		data.Namespace,
```

Comment:
there is a reason to not send the data object?

Reply:
We would need to expose the object to components/components.go. In this state we pass basic explicit values (except addons images). `GetDeployment` is also used in HCO. I'd rather keep it basic (for now).